www/www-deploy.yml

```diff
@@ -202,6 +208,20 @@
     - match: "~/site/support/ome-artwork(/.*)?$"
       dest: /artwork
 
+    # uppercase alias
+    - match: "~/BIO-FORMATS$"
+      dest: /bio-formats
+    - match: "~/BIO-FORMATS/(?<link>.*)$"
+      dest: /bio-formats/$link
+    - match: "~/OME-FILES$"
+      dest: /ome-files
+    - match: "~/OME-FILES/(?<link>.*)$"
+      dest: /ome-files/$link
+    - match: "~/OMERO$"
+      dest: /omero
+    - match: "~/OMERO/(?<link>.*)$"
```

Comment:
My primary questions was whether we need the full range of regexp handling in which case this PR is definitely the way to go or only handle capitalized versions for some defined URLS e.g. `/OMERO/` and maybe `/OMERO/downloads`. In the latter case, I would be including to use `jekyll-redirect-from` in the usptream http://github.com/openmicroscopy/www.openmicroscopy.org code for single-page internal redirect.